integration_tests/mockApis/tokenVerification.ts

```diff
@@ -0,0 +1,29 @@
+import { SuperAgentRequest } from 'superagent'
+import { stubFor } from './wiremock'
+
+export default {
+  // stubTokenVerificationPing: (status = 200): SuperAgentRequest =>
```

Comment:
Should this be commented out still?

Reply:
it comes in the next PR - I must have realised trying to test login in the same PR as getting cypress itself up and running was a touch ambitious 

https://github.com/ministryofjustice/offender-categorisation/pull/694/files#diff-78a88cd5054960f522896487b9a0a3848574cce32f10efe54a475b1b66be7ffe

Reply:
(removing)